htlcswitch/circuit_map.go

```diff
@@ -219,6 +219,11 @@ func NewCircuitMap(cfg *CircuitMapConfig) (CircuitMap, error) {
 		return nil, err
 	}
 
+	// Delete old circuits and keystones of closed channels
```

Comment:
Hmm, so we'll do this at start up rather than on an _ongoing_ basis? I guess in the end, at least with bolt, we only really shrink disk space no compaction, which (atm) itself requires a restart....

Also style nit: missing a period here.

Reply:
Period added.

I thinking on an ongoing basis, there is `DeleteCircuits` being called multiple times to clean up circuits. Haven't dug into all those call sites yet while I think most of the circuits are cleaned before the channel being closed?